sdk/python/kfp/components/_dsl_bridge.py

```diff
@@ -148,13 +149,32 @@ def _create_container_op_from_resolved_task(name:str, container_image:str, comma
     
     output_paths_for_container_op = {output_name_to_kubernetes[name]: path for name, path in output_paths.items()}
 
+    component_meta = {}
+    component_meta["name"] = component_spec.name
+    component_meta["description"] = component_spec.description
+    component_meta["inputs"] = {"parameters": []}
+    if component_spec.inputs is not None:
+        for input in component_spec.inputs:
+            component_meta["inputs"]["parameters"].append({
+                "name": input.name,
+                "description": input.description,
+                "type": input.type
```

Comment:
What about the default value? I'm not sure DSL compiler adds defaults to the template inputs.